rpc/common/moc_common_common.proto

```diff
@@ -74,8 +81,9 @@ message Version {
 message Status {
 	Health health = 1;
 	ProvisionStatus provisioningStatus = 2;
-	Error lastError = 3;
-	Version version = 4;
+	DownloadStatus downloadStatus = 3;
+	Error lastError = 4;
```

Comment:
I don't think we should change existing numbering in here for backward compatibility